WordPress/Classes/ViewRelated/Post/AbstractPostListViewController.m

```diff
@@ -305,14 +248,6 @@ - (void)configureSearchWrapper
     self.searchWrapperView.backgroundColor = [WPStyleGuide wordPressBlue];
 }
 
-- (NSDictionary *)propertiesForAnalytics
```

Comment:
Yes, it was a merge conflict I must've messed up. Adding this back in fixed the crash. Thanks for that!